src/main/resources/lang/english.lang

```diff
@@ -231,3 +231,10 @@ io exceptions:
 misc:
 	namespacedutils:
 		format: A namespaced key can be formatted as 'namespace:id' or 'id'. It can only contain one ':' to separate the namespace and the id. Only alphanumeric characters, periods, underscores, and dashes can be used.
+
+# -- Functions --
+functions:
```

Comment:
I would keep single quotes around function/parameter names. It improves readability.